src/utils/transform.js

```diff
@@ -150,6 +150,8 @@ export const getRSQuery = (componentId, props, execute = true) => {
 					featuredSuggestionsConfig: props.featuredSuggestionsConfig,
 					indexSuggestionsConfig: props.indexSuggestionsConfig,
 					enableFeaturedSuggestions: props.enableFeaturedSuggestions,
+					enableFAQSuggestions: props.enableFAQSuggestions,
```

Comment:
Conditionally pass `enableFAQSuggestions ` and `FAQSuggestionsConfig` when `searchboxId` prop is used, else you might consider showing a warning in the console.